scanamo/src/main/scala/org/scanamo/DynamoFormat.scala

```diff
@@ -215,6 +201,30 @@ object DynamoFormat extends LowPriorityFormats {
   def coercedXmap[A, B: DynamoFormat, T >: Null <: Throwable: ClassTag](read: B => A, write: A => B): DynamoFormat[A] =
     xmap(coerce[B, A, T](read), write)
 
+  implicit final def genericFormat[A](implicit U: AutoDerivationUnlocker): DynamoFormat[A] = macro deriveGenericFormat[A]
+
+  def deriveGenericFormat[A: c.WeakTypeTag](c: scala.reflect.macros.whitebox.Context)(U: c.Tree): c.Tree = {
+    val _ = U
+    Magnolia.gen[A](c)
+  }
+
```

Comment:
I think source order does matter especially with vals, it may be preferrable to move these two definitions down at the bottom.

Reply:
Done in 832120b183652b470ee07c6909505d473de2eb1f